build.assets/Makefile

```diff
@@ -204,12 +248,13 @@ buildbox-centos7-fips: buildbox-centos7
 buildbox-arm:
 	$(call PULL_ON_CI,$(BUILDBOX_ARM))
 	docker buildx build \
-		--build-arg BUILDBOX_VERSION=$(BUILDBOX_VERSION) \
```

Comment:
Was this meant to be removed? It seems unrelated to the described changes.

Reply:
Ah yes, this was an opportunistic change since I noticed that Dockerfile-arm doesn't contain any such `ARG`